packages/assets-controllers/src/TokensController.ts

```diff
@@ -960,6 +961,22 @@ export class TokensController extends BaseController<
       );
     }
   }
+
+  initApprovals() {
```

Comment:
Is it worth making this `async` to make it explicit and enforce a promise is always returned?

Reply:
Yes. I don't even know why the linter didn't let me know, since it normally warns you when adding a method that returns a Promise but is not async.